test/verify/check-journal

```diff
@@ -291,23 +307,99 @@ ExecStart=/bin/sh -c 'for s in $(seq 10); do echo SLOW; sleep 0.1; done; sleep 1
                         "Load earlier entries"
                         ])
 
-        b.go("#/?start=boot&prio=6")
+        b.go("#/?start=boot&prio=info")
 
         wait_log_present("AFTER BOOT")
         m.execute("logger -p 5 --tag just-a-service WORK PLEASE")
         wait_log_present("WORK PLEASE")
-        self.select_from_dropdown(b, "#journal-service-menu", "check-journal", click=False)
-        self.select_from_dropdown(b, "#journal-service-menu", "just-a-service", click=True)
+        if m.image == "rhel-8-2-distropkg": # Changed in #13905
+            self.select_from_dropdown(b, "#journal-service-menu", "check-journal", click=False)
+            self.select_from_dropdown(b, "#journal-service-menu", "just-a-service", click=True)
+        else:
+            b.wait_present("#journal-service-menu option[value='check-journal']")
+            b.set_val("#journal-service-menu", "just-a-service")
         wait_log_not_present("AFTER BOOT")
         wait_log_present("WORK PLEASE")
-        self.select_from_dropdown(b, "#journal-service-menu", "All", click=True)
+        if m.image == "rhel-8-2-distropkg": # Changed in #13905
+            self.select_from_dropdown(b, "#journal-service-menu", "All", click=True)
+        else:
+            b.set_val("#journal-service-menu", "*")
         wait_log_present("AFTER BOOT")
         wait_log_present("WORK PLEASE")
 
-        self.select_from_dropdown(b, "#journal-prio-menu", "Error and above")
-        self.select_from_dropdown(b, "#journal-service-menu", "check-journal", click=False)
-        b.click("#journal-service-menu button")
-        b.wait_not_present("#journal-services-list ul li a:contains('just-a-service')")
+        self.select_priority("Error and above")
+        wait_log_not_present("WORK PLEASE")
+        if m.image == "rhel-8-2-distropkg": # Changed in #13905
+            self.select_from_dropdown(b, "#journal-service-menu", "check-journal", click=False)
+            b.click("#journal-service-menu button")
+            b.wait_not_present("#journal-services-list ul li a:contains('just-a-service')")
+        else:
+            b.set_val("#journal-service-menu", "check-journal")
+            b.wait_not_present("#journal-service-menu option[value='just-a-service']")
+
+        if m.image == "rhel-8-2-distropkg": # Introduced in #13905
+            return
+
+        # Test following
+        b.go("#/?tag=test-stream&prio=notice")
+        wait_journal_empty()
+
+        m.execute("logger --tag test-stream Message 1")
+        b.wait_present("#journal-box .cockpit-logline .cockpit-log-message:contains('Message 1')")
+
+        b.click("#journal-follow:contains('Pause')")
+        b.wait_visible("#journal-follow:contains('Stream')")
```

Comment:
I demoed this to @garrett and @jgiardino today, and they suggested **Resume** might work better as a label than **Stream**, as it's a more common opposite verb to **Pause**, has clear analogues with playback in the physical world etc. Could even enforce that with the icons fa-play and fa-pause.
This is also what OpenShift does for their log streaming UI, so it's a gain for cross-product consistency.
I'll try to find a screenshot of that.